roles/openshift_node/tasks/main.yml

```diff
@@ -33,6 +33,8 @@
         local_quota_per_fsgroup: "{{ openshift_node_local_quota_per_fsgroup | default(None) }}"
         dns_ip: "{{ openshift_dns_ip | default(none) | get_dns_ip(hostvars[inventory_hostname])}}"
         env_vars: "{{ openshift_node_env_vars | default(None) }}"
+        min_tls_version: "{{ openshift_node_min_tls_version | default(None) }}"
+        cipher_suites: "{{ openshift_node_cipher_suites | default(None) }}"
```

Comment:
I was following the existing patterns - it wouldn't be necessary to persist this locally, I'll update to use inventory variables directly.